test/e2e/limitations/report_limitation_test.go

```diff
@@ -0,0 +1,274 @@
+//go:build e2e
+
+package limitations
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/anthropics/anthropic-sdk-go"
+
+	"github.com/cchalm/blundering-savant/test/e2e/testutil"
+)
+
+// TestReportLimitationInsteadOfFileWorkarounds tests that AI reports limitations instead of file workarounds
+func TestReportLimitationInsteadOfFileWorkarounds(t *testing.T) {
```

Comment:
Let's put all AI tests into a single package called `ai`. There can be multiple files, but let's do one package